frontends/infinite-corridor/src/libs/react-query.ts

```diff
@@ -1,5 +1,10 @@
 import { QueryClient } from "react-query"
 import axios from "./axios"
+import { fieldDetails } from "../api/fields/urls"
+import { FieldChannelForm } from "../api/fields/interfaces"
+
+axios.defaults.xsrfCookieName = "csrftoken"
+axios.defaults.xsrfHeaderName = "X-CSRFToken"
```

Comment:
This can go in `frontends/infinite-corridor/src/libs/axios.ts`:
```ts
/**
 * Our axios instance with default baseURL, heads, etc.
 */
const instance = axios.create({
  baseURL:        "/api/v0",
  xsrfCookieName: "csrftoken",
  xsrfHeaderName: "X-CSRFToken"
})
```